ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Issue Report
+about: Create a report to help us improve
+
+---
+
```

Comment:
😅 That looks great but its okay to remove this section as it is obvious that someone clicking on issue will surely know how to use or for what it is used!